test_models/test_model_spacy_sklearn/model_20181030-144819/entity_synonyms.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
please update all references to the `test_models` dir in the code (mostly in the tests I believe)

Reply:
what do you mean with this one?

Reply:
I mean the explicit paths in `test_utils.py`